src/scss/core/layout/_header.scss

```diff
@@ -6,8 +6,19 @@ $component-name: header;
   z-index: 100;
   width: 100%;
   max-width: 100vw;
+  background-color: $header-bg !important;
```

Comment:
I really have problem with using `!important` :)

Reply:
Me too. But ps_searchbar.css has styles that have more preference than this.
That's why I choose this approach, but maybe modify ps_searchbar will be better

Reply:
@rodriciru please do modify `ps_searchbar` if needed :-)

Reply:
The problem is, how can I remove CSS from other CSS?
In ps_searchbar.css we have this:
`#header, #header .header-top {
  background-color: white;
}`
How can I remove or delete from my SCSS file without !important

Reply:
One way was to add a ps_searchbar.css in themes/modules/ps_searchbar dir, but when the maintainers of that module add some CSS customization, we need to change in the theme, and I don't like it.

Reply:
> Me too. But ps_searchbar.css has styles that have more preference than this. That's why I choose this approach, but maybe modify ps_searchbar will be better

https://github.com/PrestaShop/ps_searchbar/pull/47 this will remove ps_serachbar's preference

Reply:
As @SharakPL mentioned, just need to wait other PR has merged.
Remove the `!important` and everything will be ok.

Btw we have a border-bottom and need to define variable for header-top and header-bottom:
.header .header__top ln:5
.header .header__bottom ln:29

Reply:
> Btw we have a border-bottom and need to define variable for header-top and header-bottom: .header .header__top ln:5 .header .header__bottom ln:29

I don't get you @mparvazi

Reply:
`src/scss/custom/layout/_header.scss`

Ln:5

`$header-top-border: $gray-200;`

Reply:
@rodriciru everything looks fine to me but can you take @mparvazi 's comment in account? It would be cool to be able to customize this border too

Reply:
Done

Reply:
BTW: Don't forget that we want to do a huge cleanup after everything is styled to somehow-finished-state.
- Reorganize
- Clean
- Move things between core/custom so the theme devs have a clean boostrap base. :-)